File: Strong-&-Brutal/tahtib/tahtib.py
# tahtib.py – A Ninja Egípcia da Tempestade

class Tahtib:
    def __init__(self):
        self.berserker = 0

    def gain_berserker(self):
        self.berserker += 1

    def doton(self):
        print("Q – Doton: Cegueira por 2s, Dano Verdadeiro contínuo, Roubo de Vida.")

    def suiton(self):
        print("W – Suiton: Lentidão por 2s, Dano Contínuo, Roubo de Vida.")

    def katon(self):
        print("E – Katon: Dano ígneo por 1s, Redução de regeneração do inimigo.")

    def raiton(self):
        print("R – Raiton (Ultimate): Dano elétrico massivo, Paralisia por 3s.")

File: Strong-&-Brutal/tahtib/Tahtib.java
// Tahtib.java – A Ninja Egípcia da Tempestade

public class Tahtib {
    private int berserker;
    
    public Tahtib() {
        this.berserker = 0;
    }

    public void gainBerserker() {
        berserker++;
    }

    public void doton() {
        System.out.println("Q – Doton: Cegueira por 2s, Dano Verdadeiro contínuo, Roubo de Vida.");
    }

    public void suiton() {
        System.out.println("W – Suiton: Lentidão por 2s, Dano Contínuo, Roubo de Vida.");
    }

    public void katon() {
        System.out.println("E – Katon: Dano ígneo por 1s, Redução de regeneração do inimigo.");
    }

    public void raiton() {
        System.out.println("R – Raiton (Ultimate): Dano elétrico massivo, Paralisia por 3s.");
    }
}
